subiquity/client/client.py

```diff
@@ -136,6 +136,8 @@ def __init__(self, opts):
 
         self.conn = aiohttp.UnixConnector(self.opts.socket)
         self.client = make_client_for_conn(API, self.conn, self.resp_hook)
+        self.client1 = make_client_for_conn(
```

Comment:
Will ubuntu-desktop-installer need to be able to hit client1?  

At some point it will want to support autoinstall, this is part of how interactive-sections works, right?

Reply:
Yeah this is all around interactive-sections really: basically the client doesn't know when the installer is ready to start curtin so there is a bunch of hair to tell it. Currently udi ignores all this though and just calls .confirm() after submitting the storage config...